### ISSUE TITLE
Uncaught TypeError: e.node.getBoundingClientRect is not a function

### ISSUE THREAD

**Issue by aap82** [mobxjs/mobx-react-devtools#80]
With displayUpdates enabled, and using a component like:

`ObjPropDisplayer = observer ({ obj, prop }) => obj?[prop] or null`

throws the above mentioned error because, well, it's not a DOM node, but still is a valid React (observer) component.

**Comment by jamespedid** [comment]
I have this issue too. The node element that is passed into the event object is text but there is no function. This should be addressed considering React can properly handle components that can simply return text.

![image](https://user-images.githubusercontent.com/1280555/39089261-da7cda18-4590-11e8-9936-d31fcedcefa2.png)

To recreate it, simply use a component like 
```
const FunctionalTextComponent = () => 'My functional text component'
```

Did not have this issue in 4.2.0

**Comment by mweststrate** [comment]
Does this still happen with mobx-react-devtools@6.0.1?

**Comment by aap82** [comment]
this
```
const Title = observer(({ obj }) => `Counter App - ${obj.count}`)
```
Still [does not work](https://stackblitz.com/edit/mobx-react-devtools-issue-80-hz4feg?file=index.js).

**Comment by mweststrate** [comment]
Fixed and released as 6.0.2

cc @aap82 @jamespedid
